vo_models/uws/models.py

```diff
@@ -108,8 +113,8 @@ class ResultReference(BaseXmlModel, tag="result", ns="uws", skip_empty=True, nsm
 class Results(BaseXmlModel, tag="results", ns="uws", nsmap=NSMAP):
     """The element returned for /{jobs}/{job-id}/results
 
-    Elements:
-    result list[ResultReference]: a list of references to UWS results.
+    Parameters:
+        results (list[ResultReference]): (element) A list of references to UWS results.
```

Comment:
this docstring doesn't include the `Optional` part to it from the type annotation